data/defaults.yaml

```diff
@@ -326,3 +326,11 @@ prometheus::grok_exporter::package_ensure: 'latest'
 prometheus::grok_exporter::package_name: 'grok_exporter'
 prometheus::grok_exporter::user: 'grok-exporter'
 prometheus::grok_exporter::version: '1.0.0.RC4'
+prometheus::openvpn_exporter::download_extension: ''
```

Comment:
hey,
instead of adding the values here? can you please directly set them in prometheus::openvpn_exporter? That's a new standard we want to establish. That enables puppet-strings to automatically pick up the values and place them into the REFERENCE.md

Reply:
Alright, just moved those defaults with my latest commit. Please let me know if you see anything else that I should change